### DISCUSSION TITLE
[Support Request] This API should follow latest version of Minecraft sooner or later then latest version of Minecraft gets released

### DISCUSSION THREAD

**Discussion by joeyoropesa-dev** [Sinytra/ForgifiedFabricAPI#85] [General]
I'm supprised basically then many mods gets updated to latest version of Minecraft release (including their modloaders) plugins like this always are lower version than the latest version.

How much time is required to keep it up-to-date with Minecraft latest version **before** Minecraft go up to the next version.

Like for example, current latest version of Minecraft is 1.20.2 and I have a feeling that when 1.20.2 support for this API port on Forge gets released, Minecraft will already move to the next latest version (build) 1.20.3

**Comment by Su5eD** [comment]
FFAPI is open source, everyone is free to contribute should they want to speed up the process. I'll appreciate any help.
